### PR TITLE
lightning: fix a heading in physical import mode

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

The original description is inappropriate because physical import mode can be used in production environment even before v6.2. 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 1]
```diff
@@ -126,11 +126,9 @@ mysql> select table_name,index_name,key_data,row_data from conflict_error_v1 lim
 
 根据上述信息人工甄别需要保留的重复数据，手动插回原表即可。
 
-## 导入数据到生产集群
+## 导入时限制调度范围从集群降低到表级别
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:129]: If this title is changed, the user doc links in 6.2.0 release notes should be updated as well.

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by shichun-0415 — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]
LGTM

**Pull request merged**